doc/source/programs/gdal_vector_rasterize.rst

```diff
@@ -0,0 +1,214 @@
+
+.. option:: --nodata <NODATA>
+
+        Assign a specified nodata value to output bands.
+
+.. option:: --init <INIT>
+
+    Pre-initialize output bands with specified value. May be repeated.
+
+.. option:: --srs <SRS>
+
+    Override the projection for the output file. If not specified, the projection of the input vector file will be used if available. When using this option, no reprojection of features from the SRS of the input vector to the specified SRS of the output raster, so use only this option to correct an invalid source SRS. The <srs_def> may be any of the usual GDAL/OGR forms, complete WKT, PROJ.4, EPSG:n or a file containing the WKT.
+
+.. option:: --transformer-option <NAME>=<VALUE>
+
+    set a transformer option suitable to pass to GDALCreateGenImgProjTransformer2(). This is used when converting geometries coordinates to target raster pixel space. For example this can be used to specify RPC related transformer options.
+
+.. option:: --target-extent <xmin> <ymin> <xmax> <ymax>
+
+    Set georeferenced extents. The values must be expressed in georeferenced units. If not specified, the extent of the output file will be the extent of the vector layers.
+
+.. option:: --target-resolution <xres> <yres>
+
+    Set target resolution. The values must be expressed in georeferenced units. Both must be positive values. Note that `--target-resolution` cannot be used with `--target-size`.
+
+.. option:: --tap
+
+    (target aligned pixels) Align the coordinates of the extent of the output file to the values of the -tr, such that the aligned extent includes the minimum extent. Alignment means that xmin / resx, ymin / resy, xmax / resx and ymax / resy are integer values.
+
+.. option:: --target-size <xsize> <ysize>
+
+    Set output file size in pixels and lines. Note that `--target-size` cannot be used with `--target-resolution`.
+
+.. option:: --ot, --datatype, --output-data-type <OUTPUT-DATA-TYPE>
+
+    Force the output bands to be of the indicated data type. Defaults to `Float64`, unless the attribute field to burn is of type `Int6`4, in which case `Int64`` is used for the output raster data type if the output driver supports it.
+
+.. option:: --optimization <OPTIMIZATION>
+
+    Force the algorithm used (results are identical). The raster mode is used in most cases and optimise read/write operations. The vector mode is useful with a decent amount of input features and optimise the CPU use. That mode have to be used with tiled images to be efficient. The auto mode (the default) will chose the algorithm based on input and output properties.
+
+Advanced options
+++++++++++++++++
+
+.. include:: gdal_options/oo.rst
+
+.. include:: gdal_options/if.rst
+
+
+Examples
+--------
+
+.. example::
+   :title: Burn a shapefile into a raster
+
+    The following would burn all polygons from :file:`mask.shp` into the RGB TIFF file :file:`work.tif` with the color red (RGB = 255,0,0).
```

Comment:
However, the result may surprise the user because the default datatype is Float64.